std/conv.d

```diff
@@ -5275,15 +5275,18 @@ unittest
     Params:
         From  = The type to cast from. The programmer must ensure it is legal
                 to make this cast.
-        To    = The type to cast to
-        value = The value to cast. It must be of type $(D From),
-                otherwise a compile-time error is emitted.
-
-    Returns:
-        the value after the cast, returned by reference if possible
```

Comment:
This actually aids readability. Please read the initial pull description as there is a similar case and a link to a preview of the generate documentation.